views/template.pug

```diff
@@ -6,7 +6,7 @@ html(lang="en")
     meta(name="viewport", content="width=device-width, initial-scale=1.0")
     meta(http-equiv="X-UA-Compatible", content="ie=edge")
 
-    title LATE
+    title #{title || "LATE"} | LATE

```

Comment:
I would change this to `title #{title ? title + ' | ': ''}LATE` so that in the event that `ctx.state.title` is not set, we end up with `LATE` and not `LATE | LATE`.